generator/road_access_generator.cpp

```diff
@@ -146,6 +146,8 @@ TagMapping const kDefaultTagMapping = {
     {OsmElement::Tag("access", "no"), RoadAccess::Type::No},
     {OsmElement::Tag("access", "private"), RoadAccess::Type::Private},
     {OsmElement::Tag("access", "destination"), RoadAccess::Type::Destination},
+    {OsmElement::Tag("access", "emergency"), RoadAccess::Type::No},
+    {OsmElement::Tag("access", "military"), RoadAccess::Type::No},
```

Comment:
So this change is more than enough for this moment. As I said below, we can't properly handle 3-arity highway types in routing.

Reply:
ah ok 👍